libs/agno/agno/db/sqlite/schemas.py

```diff
@@ -93,6 +93,38 @@
     ],
 }
 
+TRACE_TABLE_SCHEMA = {
+    "trace_id": {"type": String, "primary_key": True, "nullable": False},
+    "name": {"type": String, "nullable": False},
+    "status": {"type": String, "nullable": False, "index": True},
+    "start_time_ns": {"type": BigInteger, "nullable": False, "index": True},
+    "end_time_ns": {"type": BigInteger, "nullable": False},
+    "duration_ms": {"type": BigInteger, "nullable": False},
+    "total_spans": {"type": BigInteger, "nullable": False},
```

Comment:
Not sure about this one. It can be derived right? What if you add more spans? This is a weird one to have for a join relationship.

Reply:
updated to be at api level